test/powershell/Language/Parser/Conversions.Tests.ps1

```diff
@@ -127,3 +127,57 @@ namespace TestTypeResolution {
         $location | Should Be $cmdletDllPath
     }
 }
+
+Describe 'method conversion' -Tags 'CI' {

```

Comment:
I think you should add tests for varying number of parameters (maybe up to 10?) and also void returns, and probably find some delegates other than Func and Action.

You also need to test overloads - make sure each overload is correctly chosen.

And I would also test multiple classes - making sure we don't call the wrong method, perhaps incorrectly cached from another conversion.